packages/client/src/topics.json

```diff
@@ -34,6 +34,11 @@
       "permits": ["AkxinYRNNo679qi6T", "TuP6ido9xpJSSxjLi"],
       "slug": "zonwering-of-rolluik-plaatsen"
     },
+    {
+      "path": "imtr/transformed/formulier.json",
+      "permits": ["TuP6ido9xpJSSxjLi"],
+      "slug": "formulier"
```

Comment:
slug `formulier`?

Reply:
For demo purposes. We need to decide what the actual slug will be. Noted

Reply:
it's here: https://trello.com/c/6gxBSFYf/742-vergunning-check-vs-vergunning-aanvraag-teksten